Applications/ISIS.md

```diff
@@ -0,0 +1,74 @@
+## Introduction
+The ISIS software project is a collection of planetary image processing applications for data collected by a wide range of sensors. The source code and issue tracker are hosted on [GitHub](https://github.com/USGS-Astrogeology/ISIS3) and it's documentation is hosted on a separate [website](https://isis.astrogeology.usgs.gov/).
+
+## History & Outlook
+Originating in the 1970s, ISIS has a long history that is detailed [here](https://isis.astrogeology.usgs.gov/documents/IsisHistory/IsisHistory.html).
+
+The USGS Astrogeology Science Center (ASC) has been the owner and maintainer of ISIS. Recently, the ASC has started transitioning the ISIS software project to open source and is seeking greater community involvement. A [technical committee](https://github.com/USGS-Astrogeology/ISIS_TC) has been stood up and governance of the project is being gradually transferred from the ASC to the ISIS technical committee (ISIS TC).
+
+## Metrics
+
+Currently, the majority of code contributors are employees or past employees of the Astrogeology Science Center (ASC). In the last year, a small number of non-ASC personnel have contributed to the project. Issues and documentation have been contributed by the broader user community.
+
+## Scope
+The ISIS software project encompasses software used for photometric and photogramettric processing of planetary remote sensing data.
+
+The ISIS working group will have sole responsibility and discretion over the Planetary Software
+project in the following areas:
+
+* Release processes (including setting release cadence and release quality standards).
+* Project governance and process (including this policy).
+* Location of the official software repository.
+* Conduct guidelines.
+* Maintaining the list of contributors.
+* Contribution guidelines (including coding and testing standards).
+* Mediating technical conflicts between Collaborators.
```

Comment:
Does it make sense to link directly back to the doc where these area articulated on the ISIS-TC side? One of the things I like about condo-forge maintainers is that the person being added as a maintainer needs to +1 the PR. Should the ISIS TC +1 this PR? Or is a link back to this language sufficient?

Reply:
I think that if a proto-TC already has a such guidelines in place, then a link to their document should be sufficient (preferred, in fact over re-iterating it).

The "ISIS TC" has already submitted the PR, do they need to also +1 it?  Seems redundant.